scrapers/il/bills.py

```diff
@@ -934,3 +930,9 @@ def build_sponsor_list(sponsor_atags):
             official_spontype = "cosponsor"  # until replaced
         sponsors.append((spontype, sponsor, chamber, official_spontype))
     return sponsors
+
+
+def clean_archivebill_url(url):
+    if "https://beta.ilga.gov/documents/" not in url:
```

Comment:
Please use a `BASE_URL` constant (or similar variable) in place of the literal string `https://beta.ilga.gov`

Reply:
Fixed issue